backends/advanced/src/advanced_omi_backend/speaker_recognition_client.py

```
@@ -67,14 +67,21 @@ def __init__(self, service_url: Optional[str] = None):
             logger.info("Speaker recognition client disabled (no service URL configured)")
 
     async def diarize_identify_match(
-        self, audio_data: bytes, transcript_data: Dict, user_id: Optional[str] = None
+        self,
+        conversation_id: str,
+        backend_token: str,
+        transcript_data: Dict,
+        user_id: Optional[str] = None
```

Ruff (0.14.11)

Unused method argument: `user_id`

(ARG002)